spec/requests/api_registrations_spec.rb

```diff
@@ -428,20 +427,6 @@
       expect(email.subject).to eq(I18n.t('registrations.mailer.new.mail_subject', comp_name: registration.competition.name))
     end
 
-    it 'raises error if registration doesnt exist' do
```

Comment:
Looks like we're just deleting this test? Is it being replaced by something?

Reply:
Oh, I had intermittently deleted this because I was hell-bent on querying registrations only by numeric ID. In that case, this test would have been useless because you cannot even query a non-existing registration if you don't have a numeric ID to query for.

But now I already understood that the "competitionId+userId" tuple querying is unavoidable, so I'll happily reinstate this test.

Reply:
Nevermind: I just realized that this test was trying to do a `PATCH` against a non-existing (competitionId, userId)` tuple. That won't work anymore because the `PATCH` route now definitely needs a `registrationId`. We could send a zero and see that this blows up? Not sure whether that helps though

Reply:
Yeah I think sending a zero or any other reg value we know doesn't exist and asserting that we get a 404 back is the way to go here